airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/AirbyteConnectorRunnable.kt

```diff
@@ -40,8 +42,17 @@ class AirbyteConnectorRunnable : Runnable {
                     "Failed ${operation::class} operation execution."
                 }
             }
-            outputConsumer.accept(exceptionHandler.handle(e))
-            throw e
+            if (operationName == "check") {
```

Comment:
Required because config is created as part of the initialization of the operation now, not within the operation anymore. See PR description for more information

Reply:
My rule of thumb is that If you feel like you should add a comment on the PR, there should be a comment in the code explaining what's going on.